instaloader/instaloadercontext.py

```diff
@@ -215,8 +216,9 @@ def login(self, user, passwd):
         session.headers.update({'X-CSRFToken': csrf_token})
         # Not using self.get_json() here, because we need to access csrftoken cookie
         self.do_sleep()
+        # Workaround credits to pgrimaud. See: https://github.com/pgrimaud/instagram-user-feed/commit/96ad4cf54d1ad331b337f325c73e664999a6d066
         login = session.post('https://www.instagram.com/accounts/login/ajax/',
-                             data={'password': passwd, 'username': user}, allow_redirects=True)
+                             data={'enc_password': f'#PWD_INSTAGRAM_BROWSER:0:{calendar.timegm(time.gmtime())}:{passwd}', 'username': user}, allow_redirects=True)
```

Comment:
```suggestion
                             data={'enc_password': enc_password, 'username': user}, allow_redirects=True)
```
(use abovely-mentioned `enc_password` and fix too long line)